deploy/charts/library/templates/_cluster-role.tpl

```diff
@@ -148,4 +148,14 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
     verbs: ["get", "update", "delete", "list"]
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-default
+  namespace: {{ .Release.Namespace }} # namespace:cluster
+rules:
+  - apiGroups: [""]
+    resources: ["pod"]
```

Comment:
Why is this pod get role added? I don't expect any new privileges for the default service account.

Reply:
I can remove it

Reply:
That was just for testing